userguide/jails.rst

```diff
@@ -552,7 +552,7 @@ Click :guilabel:`NEXT` to view all jail
    |                        |              | Set to *Disable*: stop the jail from using IPv4 entirely.                                               |
    |                        |              |                                                                                                         |
    +------------------------+--------------+---------------------------------------------------------------------------------------------------------+
-   | ip6_saddrsel           | string       | Only available when the jail is not configured to use VNET. Disables IPv6 source address selection      |
+   | ip6.saddrsel           | string       | Only available when the jail is not configured to use VNET. Disables IPv6 source address selection      |
```

Comment:
Definition first. Also needs punctuation at the end of the sentence.  Whitespace probably wrong due to github.
```suggestion
   | ip6.saddrsel           | string       | Disable IPv6 source address selection. Only available when the jail is not configured to use VNET.       |
```